### DISCUSSION TITLE
How to configure for not monitoring the host (disable node-exporter)

### DISCUSSION THREAD

**Discussion by cdesch** [prometheus-operator/kube-prometheus#1381] [Q&A] [answered]
Is there a way to configure the `prometheus-operator` to only monitor the pods and namespaces and not the Host? 

For example, when launching the [`node-exporter-daemonset.yaml`](https://github.com/prometheus-operator/kube-prometheus/blob/557dbd1a099c539e44b10a7f0c13a48a755c5bde/manifests/node-exporter-daemonset.yaml#L44) I get the error:

    Error from server (Forbidden): error when creating "manifests\\node-exporter-daemonset.yaml": 
    daemonsets.apps "node-exporter" is forbidden: non-admin user "44b0d8a7-2fbb-4b60-9162- 
    f439b9ef93e6" [service account "monitoring:node-exporter"]. The configured privileged attributes 
    access for non-admin users ("[]")("[]") and for service accounts ("[]")("[]") lack required permissions to 
    use attributes [hostbindmounts hostnetwork hostpid] for resource node-exporter

This is expected because I'm are not allowing `HostBindMounts` on the cluster and I don't need the [`node-exporter`](https://github.com/prometheus/node_exporter) because I have another tool to do that separately. 

Is there a way to disable the `node-exporter` in the configurations?

**Comment by bip7686** [comment, marked as answer]
Just don't include the node exporter object.

Remove following line or similar:
```
{ ['node-exporter-' + name]: kp.nodeExporter[name] for name in std.objectFields(kp.nodeExporter) } +
```

**Comment by paulfantom** [comment]
Or hide it:
```
{
  nodeExporter+:: {},
}
```
